### ISSUE TITLE
0.8.23: server side pagination rendering is quite slow than 0.8.21 

### ISSUE THREAD

**Issue by s7dhansh** [aslagle/reactive-table#345]
Any reasons? Or any changes in configuration required?

**Comment by aslagle** [comment]
Yeah, that's definitely possible. v0.8.22 had a fix for #326 which required adding to the server-side pagination processing. If you don't need that fix I'd say stick with 0.8.21 for now.

**Comment by s7dhansh** [comment]
Was this fixed in the latest releases? There have been too many bug fixes after 21 :)

**Comment by s7dhansh** [comment]
The answer is no :(. Just checked. Can you please add an option to disable page count reactivity, at the very least? @aslagle

**Comment by s7dhansh** [comment]
PR: https://github.com/aslagle/reactive-table/pull/382
